NitroxClient/Helpers/NitroxProtobufSerializer.cs

```diff
@@ -43,6 +46,14 @@ private void AddType(Type type)
             model.Add(type, true);
             knownTypes[type] = int.MaxValue; // UWE precompiled is going to pass everything to us
             NitroxTypes[type] = int.MaxValue;
+
+            if (type.IsSubclassOf(typeof(MonoBehaviour))) // Add Nitrox MonoBehaviours to the Component whitelist
```

Comment:
This wouldn't accidentally include UWE types would it?

Reply:
Actually no, UWE types don't pass through this code at all, only NitroxModel and NitroxClient pass through this code